### ISSUE TITLE
Add clearer error message when not defining resources in a viash config.

### ISSUE THREAD

**Issue by DriesSchaumont** [viash-io/viash#148]
In viash 0.5.13, not defining a resource a resource in a viash config and using `viash run` provides the following error:

```
java.util.NoSuchElementException: head of empty list
        at scala.collection.immutable.Nil$.head(List.scala:469)
        at scala.collection.immutable.Nil$.head(List.scala:466)
        at com.dataintuitive.viash.functionality.Functionality.mainScript(Functionality.scala:88)
        at com.dataintuitive.viash.platforms.NativePlatform.modifyFunctionality(NativePlatform.scala:33)
        at com.dataintuitive.viash.config.Config$.read(Config.scala:227)
        at com.dataintuitive.viash.Main$.readConfig(Main.scala:137)
        at com.dataintuitive.viash.Main$.internalMain(Main.scala:69)
        at com.dataintuitive.viash.Main$.main(Main.scala:38)
        at com.dataintuitive.viash.Main.main(Main.scala)
``` 

A better error message should be provided in this case.

**Comment by rcannood** [comment]
Can you provide a minimal reproducible example? ;)

**Comment by Grifs** [comment]
Main issue that causes the exception was solved & a warning was added in Viash 0.5.16
